.github/workflows/ci.yml

```diff
@@ -69,12 +61,45 @@ jobs:
         toolchain: stable
         override: true
         target: ${{ env.NO_STD_TARGET }}
-    - name: Install dependencies
-      run: rustup +stable target add ${{ env.NO_STD_TARGET }}
+    - name: Install Rust targets
+      run: rustup +stable target add ${{ env.NO_STD_TARGET }} ${{ env.NO_STD_CHECK_TARGETS }}
+    - name: Build defmt for no_std targets
+      run: |
+        for target in $NO_STD_CHECK_TARGETS; do
+          cargo check --target $target
+        done
     - name: Build nRF52 examples
       working-directory: firmware/nrf52
       run: RUSTFLAGS='--deny warnings' cargo build --verbose --target ${{ env.NO_STD_TARGET }} --bins
 
+  mdbook:
+    strategy:
+      matrix:
+        rust:
+          - stable
+        os:
+          - ubuntu-latest
+          - windows-latest
```

Comment:
out of curiosity/wild guessing: are we building for ubuntu + windows bc we're only expecting additional hickups from non-unixoid systems?